modules/Bio/EnsEMBL/Hive/Beekeeper.pm

```diff
@@ -187,5 +187,27 @@ sub check_if_blocked {
     return $self->is_blocked;
 }
 
+
+=head2 toString
+
+  Example     : print $beekeeper->toString();
+  Description : Produces a string summary of properties of this beekeeper.
+  Returntype  : String
+  Exceptions  : none
+  Caller      : general
+  Status      : Stable
+
+=cut
+
+sub toString {
+    my ( $self ) = @_;
+
+    # FIXME: decide what else to add here
```

Comment:
I would add the options that the beekeeper was called with. Certainly the DB URL or reg_conf